drivers/can/Kconfig.mcp2515

```diff
@@ -0,0 +1,141 @@
+# Kconfig.mcp2515 - MCP2515 CAN configuration options
+
+#
+# Copyright (c) 2018 Karsten Koenig
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_MCP2515
+	bool "MCP2515 CAN Driver"
+	default n
```

Comment:
`n` is the implicit for `bool` symbols, so can skip the `default`. That makes it consistent, and generates clearer docs ("Implicitly defaults to n" instead of " - n if <bunch of inherited dependencies>").

There's a page with some [Kconfig tips and best practices](https://docs.zephyrproject.org/latest/tools/kconfig/index.html) by the way.

Reply:
All fixed, thanks